galaxy/wrapper/plotPCA.xml

```diff
@@ -12,15 +12,22 @@
             --plotTitle "$plotTitle"
             --plotFile "$outFileName"
             --plotFileFormat "$outFileFormat"
+            #if $outFileNameDataBool
+                --outFileNameData "$outFileNameData"
+            #end if
 ]]>
     </command>
     <inputs>
         <param name="corData" format="deeptools_coverage_matrix" type="data" label="Matrix file from the multiBamSummary or multiBigwigSummary tools"/>
         <expand macro="input_image_file_format" />
         <expand macro="plotTitle" />
+        <param name="outFileNameDataBool" type="boolean" label="Save the matrix of PCA and eigenvalues underlying the plot."/>
     </inputs>
     <outputs>
         <expand macro="output_image_file_format_not_nested" />
+        <data format="tabular" name="outFileNameData" label="${tool.name} on ${on_string}: PCA matrix">
+            <filter>outFileNameDataBool is True</filter>

```

Comment:
any reason you called it Bool? Is there a name conflict with the output variable? But then the heatmap has this issue as well?